models/forum.js

```diff
@@ -28,14 +28,48 @@ var ForumSchema = new Schema({
 		trim: true,
 		required: 'A thread needs the id of the poster'
 	},
-	imgurl: {
+	imgURL: {
 		type: String,
 		trim: true
 	},
-	comments: { // todo: validate inserted comments?
-		type: Array,
+	comments: {
+		type: [String],
 		default: []
-	}
+	},
+	edited: {
+        type: Boolean,
+        default: false
+    }
 });
+mongoose.model('forum', ForumSchema);
 
-mongoose.model('forum', ForumSchema);
\ No newline at end of file
+var CommentSchema = new Schema({
+    text: {
+    	type: String,
+		required: 'A comment needs to have a text',
+		trim: true
+	},
+	created: {
+        type: String,
+        default: new Date()
+    },
+    author: {
+        type: String,
+        required: 'A comment needs an author',
+        trim: true
+    },
+    posterID: {
```

Comment:
What is a posterID? Where is the ID retrieved from? Should it be a string an not a MongoDB ObjectId?

Reply:
posterID is an id from firebase specified by the applications. It is there so the applications can verify if the poster id matches the user id so they can edit delete etc.